docs/dataservice-read-catalog-example.md

```diff
@@ -1,8 +1,10 @@
 # Read example
 
-This example shows how to retrieve catalog metadata, partition metadata, and partition data using the HERE OLP Edge SDK C++.
+This example shows how to retrieve catalog metadata, partition metadata, and
+partition data using the HERE OLP Edge SDK C++.
 
-Before you run the example, you have to replace the placeholders in `examples/dataservice-read/example.cpp` with your app key and app secret:
+Before you run the example, log in by using your platform credentials, which you
```

Comment:
Why do you need to login?

Reply:
You need to authorize by replacing the placeholders in `examples/dataservice-read/example.cpp` with your app access key id, secret access key, catalog HRN and layer. Let me adjust this description and remove "log in by using your platform credentials" from READ and WRITE examples.